bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -855,6 +871,22 @@ spec:
           - patch
           - update
           - watch
+        - apiGroups:
+          - velero.io
+          resources:
+          - podvolumebackups
```

Comment:
Do you need as well perms granted to get /status for the resources?

Similarly to:
https://github.com/openshift/oadp-operator/pull/1649/files#diff-069ea2f824de6770c7836f0e18755a37580805128fe91bc747c41152efcea852R841-R843